code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -53,36 +53,39 @@
 		/datum/reagent/oxygen,
 		/datum/reagent/phosphorus,
 		/datum/reagent/potassium,
-		/datum/reagent/uranium/radium,
 		/datum/reagent/silicon,
 		/datum/reagent/silver,
 		/datum/reagent/sodium,
+		/datum/reagent/iron,
+		/datum/reagent/water,
+		/datum/reagent/oil,
+		/datum/reagent/uranium/radium,
 		/datum/reagent/stable_plasma,
-		/datum/reagent/consumable/sugar,
 		/datum/reagent/sulfur,
-		/datum/reagent/toxin/acid
 	)
 	var/list/t2_upgrade_reagents = list(
-		/datum/reagent/iron,
-		/datum/reagent/water,
-		/datum/reagent/fuel
+		/datum/reagent/consumable/sugar,
+		/datum/reagent/fuel,
+		/datum/reagent/toxin/acid,
```

Comment:
this seems like a very roundabout way to prevent chemists from making plastic

Reply:
wasn't the *intent* but it does have that effect, yeah
they can still make it, at least.